packages/core/src/lambda/vue/configEditor/samInvokeFrontend.ts

```diff
@@ -48,6 +48,7 @@ interface SamInvokeVueData {
     showNameInput: boolean
     newTestEventName: string
     resourceData: ResourceData | undefined
+    useDebugger: boolean
```

Comment:
this probably isn't needed either, because `launchConfig` on line 33 should already have `noDebug` field

Reply:
Yes, it is binding to the vue.